webapp/app/model/components/__init__.py

```diff
@@ -56,7 +56,7 @@ class DeclarationIncomesProps(StepFormProps):
 
 
 class SelectStmindProps(StepFormProps):
-    pass
+    prev_url: str
```

Comment:
I am wondering if we should put this somewhere more general like into a `LotseStepFormProps`. But this should not be part of this PR, but maybe we add a TODO comment?

```suggestion
    prev_url: str  # TODO place this at a more general place when implementing more StepFormProps for the LotseFlow
```